api/explorer.go

```diff
@@ -214,7 +226,8 @@ func (api *API) explorerBlocksHandler(w http.ResponseWriter, req *http.Request,
 		return
 	}
 	WriteJSON(w, ExplorerBlockGET{
-		Block: api.buildExplorerBlock(height, block),
+		Block: api.buildExplorerBlock(height, block,
+			req.FormValue("hexblock") == "true"),
```

Comment:
Sorry I didn't comment on this in the first pass. Wasn't sure what we wanted here, but after discussing with Luke, we agree that the call should return an error if `hexblock` is set to anything besides "true" or "false" - strict api reduces the  liklihood of dev mistakes, and coherent errors around incorrect input make it easier to find the source of the mistake.

Reply:
Makes sense to me and I will do it. I copied this pattern from `api/wallet.go` where it appears twice and does only look for "true". I also guessed that the `wallet.go` would be more strict/secure than an explorer module. So I'm going to suggest the wallet.go "force" field should be considered as well if you want to use this pattern for example to avoid misleading other devs into copying around or learning a bad pattern. @DavidVorick

Reply:
Can you give me an example of the new pattern you and luke want in the pre-existing source code please?  I'm looking through all the boolean flags in the system and not finding one "good" one according to your description.

Reply:
as per our discord discussion, we'll punt this to a follow-up PR